src/commands/indexer/index.ts

```diff
@@ -334,10 +344,12 @@ export default class Indexer extends HealthCheck {
           }
 
           break
+        }
+
         case EventType.TransferERC20:
-        case EventType.TransferERC721:
+        case EventType.TransferERC721: {
           const testLog = interestingTransaction.log!
-          if (testLog.data === undefined || testLog.data === null || testLog.data === '' || testLog.data === '0x') {
+          if (!testLog.data || testLog.data === '0x') {
             type = EventType.TransferERC721
             // this is ERC721
             // *** START OF TEMP CODE ***
```

Comment:
@alexanderattar  as per our conversation. Maybe add more to this comment.
```
A bridge event contains a "TransferERC721" event. Because our process handles a whole event bridge event, instead of the sub events we have to dedup them. So we make sure that this "TransferERC721" is not part of a bridge event.
```

Reply:
Thanks I'll add that